src/python/foglamp/device/server.py

```diff
@@ -18,6 +18,7 @@
 from foglamp.device.ingest import Ingest
 from foglamp.microservice_management import routes
 from foglamp.web import middleware
+from foglamp.microservice_management.service_registry.instance import Service
```

Comment:
I don't see any changes that means that this new import is needed. Is this a hangover from changes that have been backed out?